cmd/server/main.go

```diff
@@ -311,6 +311,7 @@ func realMain(ctx context.Context) error {
 		userSub.Handle("", userController.HandleIndex()).Queries("offset", "{[0-9]*?}").Methods("GET")
 		userSub.Handle("", userController.HandleCreate()).Methods("POST")
 		userSub.Handle("/new", userController.HandleCreate()).Methods("GET")
+		userSub.Handle("/import", userController.HandleImport()).Methods("GET")
```

Comment:
I think you'll want to handle `GET` and `POST` here, then switch inside the function to determine whether you want to render the form or process the input.

Reply:
I'll add  that  in the PR that implements the  import functionality for-reals